pkg/volume/glusterfs/glusterfs.go

```diff
@@ -819,6 +840,7 @@ func (p *glusterfsVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 		"kubernetes.io/createdby":           heketiAnn,
 		glusterTypeAnn:                      "file",
 		"Description":                       glusterDescAnn,
+		v1.MountOptionAnnotation:            "auto_unmount",
```

Comment:
thanks for the info.

@jsafrane @gnufied any comment on setting a default mount option annotation? The context of this options is to address https://bugzilla.redhat.com/show_bug.cgi?id=1424680